src/main/daml/Daml/Finance/Settlement/Instruction.daml

```diff
@@ -143,10 +143,10 @@ template Instruction
           CreditReceiver -> do
             mustAuthorize $ S.singleton routedStep.custodian
             pure CreditReceiver
-          SettleOffLedger -> do
+          SettleOffledger -> do
```

Comment:
Since this is written _off-ledger_ in the comment above, I think it is clearer with `SettleOffLedger`, but I am fine either way.

Reply:
actually I asked the Open AI chat bot:-)

```
> but you don't write it apart, like off ledger?
> Correct, "off ledger" would not be the correct way to write the term. As I mentioned earlier, "offledger" or "off-ledger" are the accepted ways to write this term. Writing it as "off ledger" would not convey the same meaning, as the words "off" and "ledger" would not be connected in a way that indicates they are used together to refer to a specific concept. In general, when two words are used together to form a single concept or idea, they should be written together in some way, either by joining them with a hyphen or by writing them as a single word. This helps to make the meaning clear and ensures that the words are read and understood correctly.
```